src/jrd/dpm.epp

```diff
@@ -245,47 +245,52 @@ double DPM_cardinality(thread_db* tdbb, jrd_rel* relation, const Format* format)
 
 	ULONG recordCount = 0, recordLength = 0;
 
-	const RelationPages* const relPages = relation->getPages(tdbb);
+	RelationPages* const relPages = relation->getPages(tdbb);
 	const vcl* const vector = relPages->rel_pages;
 	if (vector)
```

Comment:
It looks like `vector` is not used anymore, so maybe just `if (relPages->rel_pages)` ?
Not sure this check is needed at all, BTW.